packages/contracts-bedrock/medusa.json

```diff
@@ -55,7 +55,10 @@
 				]
 			},
 			"targetFunctionSignatures": [],
-			"excludeFunctionSignatures": []
+			"excludeFunctionSignatures": [
+        "ProtocolProperties.test_ERC20external_transferToZeroAddress()",
+        "ProtocolProperties.test_ERC20external_transferFromToZeroAddress(uint256)"
```

Comment:
this isn't an assumption somewhere else in the codebase, right (like in a mint or burn or smth)?

Reply:
no, it's not checked anywhere else and I didn't find it in the docs either. Perhaps it's worth checking with @0xParticle :

- [ ] Is it okay for transfers to the zero address to succeed?
- [ ] Is it okay for transferFroms to the zero address to succeed?
- [ ] and, is it okay for the above to update the balance of address(0), so it could be non-zero?

the above implies there will *not* be a bijection between `OptimismSuperchainERC20` `Transfer({from: somebody, to: address(0), amount: something})` events and `sendERC20` calls